### PR TITLE
Remove versioneer

### PR DESCRIPTION
**Pull request by willingc**

### CODE REVIEW

**Comment by willingc** [comment]
As of commit 6e1c6e3, all the Python tests and mypy stuff passes. Recommending merging this and then doing a second PR to handle the JS packaging for Circle CI and integration tests.

**Comment by willingc** [comment]
Adding back @mpacer's changes in an earlier PR solved the last puzzle piece. Yay M.

File: bookstore/_version.py [hunk 1]
```diff
@@ -1,543 +1,17 @@
...
+    "dev",  # dev or nothing
+)
+
+# pep 440 version: no dot before beta/rc, but before .dev
+# 0.1.0rc1
+# 0.1.0a1
+# 0.1.0b1.dev
+# 0.1.0.dev
+
+__version__ = ".".join(map(str, version_info[:3])) + ".".join(version_info[3:])
```
Comment (rgbkrk) [inline bookstore/_version.py]: Works for me. I'm assuming the next step is something like bumpversion?

Comment (willingc) [reply bookstore/_version.py]: Min and I just have that as part of the checklist and manually bump. That said, Python doesn't release as frequently as JS so we can add that if needed.

**Pull request merged**